docs/quick-start.md

```diff
@@ -733,32 +733,31 @@ Make sure to use the same Host header you defined in your `VMAuth` config, or th
 
 ## Next steps
 
-That's it. We obtained a monitoring cluster corresponding to the target topology:
+Now that you have a working setup with [storage](https://docs.victoriametrics.com/operator/quick-start/#storage), [scraping](https://docs.victoriametrics.com/operator/quick-start/#scraping), [alerting](https://docs.victoriametrics.com/operator/quick-start/#alerting), and secure [access](https://docs.victoriametrics.com/operator/quick-start/#access),
+and a solid understanding of how everything fits together,
+you’re ready to take the next steps with VictoriaMetrics on Kubernetes.
 
-![Cluster Scheme](quick-start_cluster-scheme.webp)
```

Comment:
I don't see this image to be re-used in the change. Shall we drop it from version control too?

Reply:
I’ve decided to keep them, as they might be linked from external sources beyond our control. Do you think it's not worse it ?

Reply:
Removed!